scan/scan_common.go

```diff
@@ -0,0 +1,78 @@
+package scan
+
+var (
+	// Network is the default network to use.
+	Network = "tcp"
+	// Short removes long/expensive tests when set.
+	Short bool
+)
+
+// Grade gives a subjective rating of the host's success in a scan.
+type Grade int
+
+const (
+	// Bad describes a host with serious misconfiguration or vulnerability.
+	Bad Grade = iota
+	// Legacy describes a host with non-ideal configuration that maintains support for legacy clients.
+	Legacy
+	// Good describes host performing the expected state-of-the-art.
+	Good
+)
+
+// Output is the result of a Scan, to be stored for potential use by later Scanners.
+type Output interface{}
+
+// Scanner describes a type of scan to perform on a host.
```

Comment:
One thing I'm trying to move to in CFSSL is to make sure structure fields are documented as well.